README.md

```diff
@@ -15,30 +15,54 @@ This repository contains the frontend and backend code for CovEd's revamped webs
 ### 🐑 Install dependencies
 To install dependencies run the following.
 
-```
+```bash
 $ npm install
 $ npx lerna bootstrap
 ```
 
+#### Dependency Errors
```

Comment:
Rename this to "Adding New Dependencies."

Reply:
Done